docs/config/index.rst

```diff
@@ -216,17 +224,15 @@ Upgrading ``astropy``
 ---------------------
 
 Each time you upgrade to a new major version of ``astropy``, the
-configuration parameters may have changed.
+configuration parameters may have changed. If you want to create a new
```

Comment:
Nothing bad happens because the default values will be used for configuration items that are not in the file. Same as now since config items are commented by default. So the only thing that will happen is that users may have an old configuration file from an old version, and thus they may miss some configuration items that will be added in or after 4.3. But we don't add config item so often, and the documentation is there to explain how to update this file if needed.

Reply:
Well, I guess it is not clear to me whether the doc is meant for end-user or downstream package maintainers, but maybe it is just me.

Reply:
Right, but for the configuration system I'm not sure if downstream packages have other documentation, I guess astropy documentation is the reference one.